roxie/ccd/ccdfile.cpp

```diff
@@ -909,9 +909,20 @@ class CRoxieFileCache : implements IRoxieFileCache, implements ICopyFileProgress
                     break;
                 if (traceLevel>8)
                     DBGLOG("Cache info dump");
+
                 // Note - cache info is stored in the DLLSERVER persistent area - which we should perhaps consider renaming
-                const char* dllserver_root = getenv("HPCC_DLLSERVER_PATH");
-                assertex(dllserver_root != nullptr);
+                StringBuffer cacheRootDirectory;
+                if (isContainerized())
+                {
+                    if (!getConfigurationDirectory(nullptr, "query", nullptr, nullptr, cacheRootDirectory))
+                        throwUnexpected();
+                }
+                else
+                {
+                    const char* dllserver_root = getenv("HPCC_DLLSERVER_PATH");
```

Comment:
Feels like this should be retrieved via environment directories section...

Reply:
I think you have left this code in so that it behaves the same as previously (i.e. only saves the cache info in bare-metal if environment var is set up (which it will not normally be). However, since that feature in bare metal is undocumented, untested, and unused (It may cause Roxie to randomly core priot to 8.2.x so I'm pretty sure no-one is relying on it) it would make sense to change the env var name to something more meaningful at this point?